### PR TITLE
#317: Button group

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-design/issues/317 |
| Dependencies | <del>https://github.com/ripe-tech/ripe-components-react-native/pull/190</del> |
| Decisions | Toggle buttons component |
| Animated GIF | toggle-buttons-iostoggle-buttons-android|

### CODE REVIEW

File: react/components/organisms/toggle-buttons/toggle-buttons.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            onUpdateValue: () => {},
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            valueData: props.active
```
Comment (NFSS10) [inline react/components/organisms/toggle-buttons/toggle-buttons.js]: Typo here

File: react/components/organisms/toggle-buttons/toggle-buttons.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            items: [],
+            value: undefined,
+            loading: false,
+            disabled: false,
+            onUpdateValue: () => {},
```
Comment (NFSS10) [inline react/components/organisms/toggle-buttons/toggle-buttons.js:28]: Add default arguments here so that this interface is properly documented

**Review by NFSS10 — CHANGES_REQUESTED** [review]

**Review by NFSS10 — APPROVED** [review]

File: react/components/organisms/toggle-buttons/toggle-buttons.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin } from "../../../util";
+
+import { ToggleButton } from "../../molecules";
+
+export class ToggleButtons extends mix(PureComponent).with(IdentifiableMixin) {
```
Comment (gcandal) [inline react/components/organisms/toggle-buttons/toggle-buttons.js:10]: We've used `ButtonGroup` in the past (https://ripe-twitch-ui-git-master-platforme.vercel.app/storybook?path=/story/components-molecules-button-group--button-group) and I feel it sounds more clear. WDYT?

Comment (BeeMargarida) [reply react/components/organisms/toggle-buttons/toggle-buttons.js]: Also renamed toggle-button to button-toggle, I forget in the previous PR

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**